core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -230,7 +236,8 @@ public synchronized void connect() throws AlluxioStatusException {
             .setClientType(getServiceName())
             .build();
         // Create stub for version service on host
-        mVersionService = ServiceVersionClientServiceGrpc.newBlockingStub(mChannel);
+        mVersionService = ServiceVersionClientServiceGrpc.newBlockingStub(mChannel)
+            .withDeadlineAfter(mRpcCallDeadlineDurationMs, TimeUnit.MILLISECONDS);
```

Comment:
@ZhuTopher For 2;
Here's how that `long` value in milliseconds is supplied to `.withDeadlineAfter(long, TimeUnit)` method.
We use milliseconds within the code as convention.